libs/ui/inputs/src/lib/autocomplete/autocomplete.component.html

```diff
@@ -2,7 +2,7 @@
   <input
     #searchInput
     type="text"
-    class="appearance-none focus:outline-white leading-tight rounded w-full text-main shadow-primary-light focus:shadow-primary"
+    class="appearance-none focus:outline-white leading-tight rounded w-full text-title shadow-primary-light focus:shadow-primary"
```

Comment:
I would put just `black` here, if text-title is green, not sure we want it green there